config.toml

```diff
@@ -16,6 +16,6 @@ motor_driving_timeout = 10000
 
 [hardware]
 n_floors = 4
-driver_address = "localhost"
-driver_port = 15657
+driver_address = "host.docker.internal"
```

Comment:
This shouldn't be changed to simulator values, the code must run on the elevators.